curriculum/challenges/english/20-upcoming-python/learn-classes-and-objects-by-building-a-sudoku-solver/6569fe0fe5b5425a1bb1f534.md

```diff
@@ -0,0 +1,137 @@
+---
+id: 6569fe0fe5b5425a1bb1f534
+title: Step 79
+challengeType: 20
+dashedName: step-79
+---
+
+# --description--
+
```

Comment:
The puzzle to solve should be given at the beginning of the project in my opinion.

Reply:
Done in step 5.